lib/transformations/resolve/resolve.js

```diff
@@ -0,0 +1,46 @@
+// Press ctrl+space for code completion
+export default function transformer(file, api) {
+	const j = api.jscodeshift;
+  
```

Comment:
Also for context on what I'm doing here, refer to other codemods like [jest-codemods](https://github.com/skovhus/jest-codemods)